src/core/text/TextMetrics.js

```diff
@@ -288,6 +289,36 @@ export default class TextMetrics
 
         return properties;
     }
+
+    /**
+     * Set custom params for calculate font metrics
+     * If validateString or baselineSymbol is not the same as a previous values than font metrics cache will reset
+     *
+     * @static
+     * @param {string} validateString - Symbols with different metrics: the highest symbol, the lowest, the widest,...
+     * @param {string} baselineSymbol - The widest symbol in fonts
```

Comment:
improve doc a little to state that this is used to help measure the height of a line of text